x-pack/plugins/fleet/cypress/screens/fleet_outputs.ts

```diff
@@ -84,6 +84,17 @@ export const loadESOutput = () =>
     hosts: ['https://bla.co'],
   });
 
+export const loadRemoteESOutput = () =>
+  loadOutput({
+    name: 'remote_es',
+    type: 'remote_elasticsearch',
+    is_default: false,
+    is_default_monitoring: false,
+    hosts: ['https://bla.co'],
+    secrets: { service_token: 'token' },
+    preset: 'balanced',
```

Comment:
strangely leaving the preset out had the effect of the secret not being created

Reply:
Hmm interesting. Any further indication why that might have happened? Might be worth experimenting with or exploring further 🤷